config/templating.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Knp\Bundle\TimeBundle\Templating\Helper\TimeHelper;
+
+return static function (ContainerConfigurator $container): void {
+    $container
+        ->services()
+            ->set('time.templating.helper.time', TimeHelper::class)
+                ->args([service('time.datetime_formatter')])
+                ->deprecate('knplabs/knp-time-bundle', '1.21', 'The "%service_id%" service is deprecated and will be removed in 2.0.')
```

Comment:
WDYT about removing the deprecations and bumping to a 2.0 right now? There wouldn't be a smooth BC path of upgrading, removing deprecations, etc, but the changes overall would be minimal (removal of the meaningless DIC parameters, some minor service ID changes (in `twig.php`)) and seem easy enough to document in a changelog.

Also, we could bump the min php version to, say, 8.1